citi-onboarding/hacka-pride
# feature(apiContato): add automatic email

## What I did
* email contact server updates

## How to test
1. Download `Insomnia core` ans install the dependencies `yarn add nodemailer express dotenv`
2. Fill the `.env` variables: `EMAIL:` will be sending e-mails from submitted forms. `PASSWORD`: password of the automated email. `EMAIL_CLIENT`: gonna recieve the emails
3. Open Insomnia select `POST` method and `JSON` structure.
4. Beside the POST method fill with `localhost:3001/api/contato`
5. The JSON structure must have the fields `name`, `email`, `subject` and `message`
6. Click on `send`. If it works you must see `Email enviado` on the screen
7. Check your real `EMAIL_CLIENT`, if you recieve an email from `EMAIL` filled, it works
